pkg/collector/process.go

```diff
@@ -351,15 +360,15 @@ func (cp *CollectingProcess) updateAddress(address net.Addr) {
 }
 
 // getMessageLength returns buffer length by decoding the header
-func getMessageLength(msgBuffer *bytes.Buffer) (int, error) {
+func getMessageLength(msgBuffer *bytes.Buffer) (int, uint16, error) {
```

Comment:
We can name the function `getMsgLenAndVersion` to be more clear

Reply:
Changed the function to get only the length.